src/controllers/userController.js

```diff
@@ -14,31 +16,20 @@ const { User } = models;
    */
 const signUp = async (req, res) => {
   const {
-    firstName, lastName, username, email, password
+    firstName, lastName, email, password
   } = req.body;
 
-  const rows = await User.findAll({
-    where: {
-      [Op.or]: [{ email }, { username }]
-    }
-  });
+  const rows = await User.findAll({ where: { email } });
```

Comment:
why are you using findAll when you're just looking for one user with that email?

Reply:
Thank you for the observation, would correct it to findOne